app/components/support_interface/reasons_for_rejection_table_component.rb

```diff
@@ -0,0 +1,30 @@
+module SupportInterface
+  class ReasonsForRejectionTableComponent < ViewComponent::Base
+    include ViewHelper
+    attr_accessor :reasons
+
+    TOP_LEVEL_REASONS_I18N_KEYS = {
```

Comment:
It might be worth making the direction of the mapping a bit more obvious, something like `TOP_LEVEL_REASONS_TO_I18N_KEYS`. Had to double check a few times which one was which.